.github/workflows/macos.yml

```diff
@@ -0,0 +1,21 @@
+name: macos
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-macos:
+    runs-on: macos-latest
```

Comment:
Can an argument be made to lock this to `macos-13` for now, and then deliberately bump it when a new version is released? I'm always a bit wary of `*-latest` runners since a future rollout of later OS runners may lead to unexpected failures.